holmes/plugins/toolsets/datadog/toolset_datadog_logs.py

```
@@ -0,0 +1,276 @@
+from enum import Enum
+import json
+import logging
+from typing import Any, Optional, Dict, Tuple
+from holmes.core.tools import (
+    CallablePrerequisite,
+    ToolsetTag,
+)
+from tenacity import RetryError
+from pydantic import AnyUrl, BaseModel, Field
```

Ruff (0.12.2)

`pydantic.AnyUrl` imported but unused

Remove unused import: `pydantic.AnyUrl`

(F401)